HUD/SetFloatingHelpTextStyle.md

```diff
@@ -6,14 +6,64 @@ aliases: ["0x788E7FD431BD67F1"]
 
 ```c
 // 0x788E7FD431BD67F1 0x97852A82
-void SET_FLOATING_HELP_TEXT_STYLE(int hudIndex, int p1, int p2, int p3, int p4, int p5);
+void SET_FLOATING_HELP_TEXT_STYLE(int hudIndex, int style, int hudColor, int alpha, int arrowPosition, int boxOffset);
 ```
 
 ## Parameters
-* **hudIndex**: 
-* **p1**: 
-* **p2**: 
-* **p3**: 
-* **p4**: 
-* **p5**: 
+* **hudIndex**: The hud index for the floating help message
+* **style**: Value 0 won't show an arrow at all. Values 1, 2 and -2 will display an arrow.
+* **hudColor**: https://docs.fivem.net/docs/game-references/hud-colors/
+* **alpha**: Value for the help box opacity, from 0-255. Anything greater will simply ignore the alpha value. Always 191 in R* scripts.
+* **arrowPosition**: Used to set the arrow positon. No value will hide the arrow
+* **boxOffset**: Offset for the floating help box. Note: Arrow stays fixed
 
+### Arrow Positions
+* 0 = Off / No arrow
+* 1 = Top
+* 2 = Left
+* 3 = Bottom
+* 4 = Right
+
+### Note
+Any numeric value greater than 4 will result in a right arrow (Index 4)
+
+### Important
+Needs to be called every frame
+
+## Examples
+```lua
+function DisplayHelpText(string)
+    BeginTextCommandDisplayHelp("STRING")
+    AddTextComponentSubstringPlayerName(string)
+    EndTextCommandDisplayHelp(1, false, false, 0)
+end
+
+CreateThread(function()
+    while true do
+        Wait(0)
+
+        local Ped = PlayerPedId()
+
+        DisplayHelpText('Example Text')
+        SetFloatingHelpTextStyle(0, 2, 2, 0, 3, 0)
+        SetFloatingHelpTextToEntity(0, Ped, 0, 0)
+    end
+end)
+```
+
+```js
+function DisplayHelpText(string) {
+    BeginTextCommandDisplayHelp("STRING")
+    AddTextComponentSubstringPlayerName(string)
+    EndTextCommandDisplayHelp(1, false, false, 0)
+}
+
+setTick(() => {
+    const Ped = PlayerPedId()
+
+    DisplayHelpText('Example Text')
+    SetFloatingHelpTextStyle(0, 2, 2, 0, 3, 0)
+    SetFloatingHelpTextToEntity(0, Ped, 0, 0)
+})
+```
+![Preview of the example above](https://i.ibb.co/G97hPn7/Image.png)
```

Comment:
This too, above `## Parameters`.

Reply:
Shouldn't this also be on a more well-known website like imgur?

Reply:
> Shouldn't this also be on a more well-known website like imgur, or even upload the image to the forums somewhere.

It should, yes. Not sure how long that site has been around (the one the author linked).